README.md

```diff
@@ -1 +1,6 @@
-# Team1_Frontend
\ No newline at end of file
+# Team1_Frontend Application
+
+## How to Run Application:
+1. Run `npm install` to build your application
+2. Start application with `API_URL=http://localhost:8080/ && npm run dev`
```

Comment:
Instruct it to add it as part the machine environment variables ie 
MacOS users add "export API_URL=http://localhost:8080/" to their .zshrc file